TvMaterialCatalog/app/build.gradle.kts

```diff
@@ -50,6 +50,8 @@ dependencies {
     implementation(libs.core.ktx)
     implementation(libs.appcompat)
     implementation(platform(libs.compose.bom))
+    implementation(libs.ui)
```

Comment:
I think we could remove these two newly added lines as Compose 1.7.0 is a part of Compose BOM now.